src/EdFi.AnalyticsMiddleTier.Tests/.env.example.docker

```diff
@@ -0,0 +1,5 @@
+MSSQL_PID = Express
```

Comment:
Shouldn't this file look similar to `\src\EdFi.AnalyticsMiddleTier.Tests\.env.example` ?

Reply:
Agreed. It needs to have the same _values_ as the docker `.env` file, but the _keys_ should come from `...Tests/.env.example`.